postgresql-11.5/src/bin/psql/commandbteq.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * psql - the PostgreSQL interactive terminal
+ *
+ * Copyright (c) 2000-2018, PostgreSQL Global Development Group
+ *
+ * src/bin/psql/command.h
+ */
+#ifndef COMMAND_H
+#define COMMAND_H
```

Comment:
Update the definition, I wonder this is going not to work properly.

Reply:
Done